quantum/dip_switch.c

```diff
@@ -34,7 +34,21 @@
 
 #ifdef DIP_SWITCH_PINS
 #    define NUMBER_OF_DIP_SWITCHES (sizeof(dip_switch_pad) / sizeof(pin_t))
+#    ifndef DIP_SWITCH_INVERT
+#       define DIP_SWITCH_INVERT {0}
+#    endif
 static pin_t dip_switch_pad[] = DIP_SWITCH_PINS;
+static uint8_t dip_switch_invert[NUMBER_OF_DIP_SWITCHES] = DIP_SWITCH_INVERT;
```

Comment:
Because of the AVR issue the code which uses `dip_switch_invert` would need to be inside a preprocessor conditional; the variable definition then should also become conditional (otherwise the compiler will complain about an unused `static` variable).